src/css/popup.css

```diff
@@ -535,18 +605,42 @@ sign-up-panel::after {
   color: var(--colorBlack)
 }
 
+.dashboard-stats-list.phones-stats li:first-child {
+  font-weight: lighter;
+}
+
 .dashboard-stats-list li:not(:first-child){
   padding-left: var(--spacingLg);
   color: var(--colorGrey50)
 }
 
+/* Phones dashboard stats styling */
+.dashboard-stats-list.phones-stats li {
+  padding-left: 0; 
+  color: var(--colorBlack);
+}
+
+.dashboard-info.forwarded-calls-texts,
+.dashboard-info.blocked-calls-texts {
+  padding-left: var(--spacingLg) !important; 
```

Comment:
question (non-blocking): Should we comment why we're using `!important`? Do we need this or can we increase specificity?

Reply:
The padding there was getting overridden by `.dashboard-stats-list.phones-stats li { padding-left: 0; ... `, so I added the `!important`